src/Microsoft.OData.Edm/EdmUtil.cs

```diff
@@ -322,16 +324,20 @@ internal static string FullyQualifiedName(IEdmVocabularyAnnotatable element)
                             }
                         }
                     }
-                    else
+                    else if (parameter != null)
                     {
-                        IEdmOperationParameter parameter = element as IEdmOperationParameter;
-                        if (parameter != null)

```

Comment:
The changes here is straightforward to consider the "EnumMember" scenario, same as other two scenarios: (property, parameter).

For we have the following sub-items:

1. Property, the full name is "fullTypeName/propertyName"
2. Parameter, the full name is  "qualified operation name/parameterName".
3. New added for EnumMember, the full name is "fullEnumTypeName/EnumMemberName".